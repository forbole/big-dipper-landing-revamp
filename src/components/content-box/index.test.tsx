import React from 'react';
import { render, screen } from '@testing-library/react';
import { MockTheme } from '@mocks';
import Component from '.';

// ==================================
// unit tests
// ==================================
describe('ContentBox', () => {
  it('matches snapshot', async () => {
    const { container } = render(
      <MockTheme>
        <Component>
          <div>HELLO WORLD</div>
        </Component>
      </MockTheme>,
    );
    expect(screen.getByText(
      /HELLO WORLD/i,
    )).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
});
