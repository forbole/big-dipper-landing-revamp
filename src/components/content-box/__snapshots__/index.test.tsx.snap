// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBox matches snapshot 1`] = `
<div>
  <div
    class="makeStyles-root"
  >
    <div>
      HELLO WORLD
    </div>
  </div>
</div>
`;
